HUD/N_0x4895bdea16e7c080.md

```diff
@@ -9,6 +9,13 @@ aliases: ["ENABLE_DEATHBLOOD_SEETHROUGH"]
 void _0x4895BDEA16E7C080(cs_type(BOOL) int p0);
 ```
 
+This native updates instructional buttons in the pause menu after contexts have been toggled.
+
+```lua
```

Comment:
Examples go into their own section; see the [contribution guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md).